modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ObjectEntryFooter.tsx

```diff
@@ -35,6 +37,19 @@ export default function ObjectEntryFooter({
 						<ClayDropDown.ItemList>
 							<ClayDropDown.Item
 								onClick={() => {
+									const hiddenInput = document.getElementById(
+										`${portletNamespace}scheduleContainer`
+									) as HTMLInputElement;
+
+									const currenthiddenInputValue = JSON.parse(
```

Comment:
Please @larissacribeiro, what do you think about add an if here, like that:

`if(hiddenInput.value)`